### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by yejia-stripe**
Codegen for openapi v197.
cc @stripe/api-libraries

## Changelog
* Change type of `ChargePaymentMethodDetailsCardPresentIncrementalAuthorizationSupported` and `ChargePaymentMethodDetailsCardPresentOvercaptureSupported` from `nullable(boolean)` to `boolean`
* Add support for `Created` on `CheckoutSession`
* Add support for `SetupFutureUsage` on `PaymentIntentConfirmPaymentMethodOptionsPixParams`, `PaymentIntentPaymentMethodOptionsPixParams`, and `PaymentIntentPaymentMethodOptionsPix`
* Deprecate `CheckoutSessionSubscriptionDataTransferDataParams.items` and `CheckoutSessionSubscriptionDataItemParams` (use the `line_items` param instead). This will be removed in the next major version.

### CODE REVIEW

File: creditnote.go [hunk 1]
```diff
@@ -61,10 +61,12 @@ type CreditNoteLineParams struct {
 // its amount_due. For a status=paid invoice, a credit note does not affect its amount_due. Instead, it can result
 // in any combination of the following:
 //
+//
 // Refund: create a new refund (using refund_amount) or link an existing refund (using refund).
 // Customer balance credit: credit the customer's balance (using credit_amount) which will be automatically applied to their next invoice when it's finalized.
 // Outside of Stripe credit: record the amount that is or will be credited outside of Stripe (using out_of_band_amount).
 //
+//
```
Comment (anniel-stripe) [inline creditnote.go:69]: Nit: Not sure why these spaces were added

File: financialconnections_account.go [hunk 2]
```diff
@@ -78,16 +78,16 @@ const (
...
 // If `category` is `credit`, one of:
 //
-//   - `mortgage`
-//   - `line_of_credit`
-//   - `credit_card`
-//   - `other`
+//  - `mortgage`
+//  - `line_of_credit`
+//  - `credit_card`
+//  - `other`
```
Comment (anniel-stripe) [inline financialconnections_account.go:90]: Same here, formatting

File: reporting_reportrun.go [hunk 3]
```diff
@@ -7,9 +7,8 @@
 package stripe
 
 // Status of this report run. This will be `pending` when the run is initially created.
-//
-//	When the run finishes, this will be set to `succeeded` and the `result` field will be populated.
-//	Rarely, we may encounter an error, at which point this will be set to `failed` and the `error` field will be populated.
+//  When the run finishes, this will be set to `succeeded` and the `result` field will be populated.
+//  Rarely, we may encounter an error, at which point this will be set to `failed` and the `error` field will be populated.
```
Comment (anniel-stripe) [inline reporting_reportrun.go:11]: Here too

**Review by anniel-stripe — APPROVED** [review]

File: creditnote.go (hunk 1 above)
Comment (yejia-stripe) [reply creditnote.go:69]: Looks like this is just reverting changes from #1547

**Pull request merged**